kratos/modified_shape_functions/triangle_2d_3_modified_shape_functions.h

```diff
@@ -173,6 +201,26 @@ class KRATOS_API(KRATOS_CORE) Triangle2D3ModifiedShapeFunctions : public Modifie
         std::vector<Vector> &rNegativeSideInterfaceAreaNormal,
         const IntegrationMethodType IntegrationMethod) override;
 
+    /**
+    * Returns the positive side outwards area normal vector values for the Gauss pts. of given quadrature.
+    * @return rPositiveExteriorFaceAreaNormal: Outwards area normal vector list.
+    * @param IntegrationMethod: Desired integration quadrature.
```

Comment:
you miss the FaceId in the parameters list

Reply:
Done!